js/project-page-functions.js

```diff
@@ -136,7 +136,7 @@ function processProjectJson(project, $revisions) {
         var dateStr = date.toLocaleString("en-US", options);
 
         if (commit.id !== myCommitId) {
-            dateStr = '<a href="../' + commit.id + '/index.html">' + dateStr + '</a>';
+            dateStr = '<a href="../' + commit.id + '/index.html" onclick="_StatHat.push(["_trackCount", "pQvhLnxZPaYA0slgLsCR7CBPM2NB", 1.0]);">' + dateStr + '</a>';
```

Comment:
@richmahn Will this work?  I'm not sure if the href and onclick will both activate when the link is pressed.

Reply:
As long as it doesn't return false in the onClick, it should be ok.

Reply:
@jag3773 See this jsfiddle to see the alert and then page change: https://jsfiddle.net/6xrm8aw5/